pcmanfm need a binary xml-purge(from its own source code) to optimize size of
data file, which cause build error on cross-compile environment. Simplely
remove it to work aound this issue.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [embedded specific]

Index: pcmanfm-0.9.7/data/ui/Makefile.am
===================================================================
--- pcmanfm-0.9.7.orig/data/ui/Makefile.am	2010-07-29 15:10:18.000000000 +0800
+++ pcmanfm-0.9.7/data/ui/Makefile.am	2010-07-29 15:11:24.000000000 +0800
@@ -21,4 +21,4 @@
 # Purge GtkBuilder UI files
 %.ui: %.glade
 	cp $< $@
-	$(top_builddir)/src/xml-purge $@
+#	$(top_builddir)/src/xml-purge $@
